metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/BusinessMetadataRules.java

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.common.server.usermetadata;
```

Comment:
If this is business metadata why are we putting it in the `usermetadata` package? or should this package just be renamed `metadata`... just adds to the confusion between the 3 types to keep overloading the terms/locations imo

Reply:
I agree to rename it as metadata package. I think the 'userMetadata' is introduced to separate from 'metadata'. We can talk about it.